### DISCUSSION TITLE
v0.10.0

### DISCUSSION THREAD

**Discussion by arxanas** [arxanas/git-branchless#1423] [Release notes]
To install or update `git-branchless`, run the following:

```console
$ cargo install --locked git-branchless
$ git branchless init  # in your repository
```

## Release highlights

This is primarily a bugfix release:

- Supports Git v2.46's reference-transaction updated format (thanks to @jblebrun).
- Supports Git's index format v4 (thanks to @samueltardieu).
- Fixes to the interactive commit selector (see [`scm-record` v0.4.0](https://github.com/arxanas/scm-record/releases/tag/v0.4.0)).

### Added

- (#1355): `git submit` now supports the `--jobs` argument for parallelism.

### Changed

- `git2` upgraded to [v0.19.0](https://github.com/rust-lang/git2-rs/releases/tag/git2-0.19.0).
- `libgit2` upgraded to [v1.8.1](https://github.com/libgit2/libgit2/releases/tag/v1.8.1).
- `scm-record` upgraded to [v0.4.0](https://github.com/arxanas/scm-record/releases/tag/v0.4.0).

### Fixed

- (#1322): Fixed the processing of symbolic refs in reference-transaction lines (since Git v2.46+).
- (#1353): `git submit` with the Phabricator forge now ignores untracked working copy changes.
- (#1363): git-branchless now supports index version 4 (via `libgit2` upgrade).
- (#1393): Branches with multivars in their configuration can now be deleted (via `libgit2` upgrade).

## Contributors

Thanks to everyone who made this release happen!

* @arxanas
* @jblebrun made their first contribution in https://github.com/arxanas/git-branchless/pull/1322
* @niklaskorz made their first contribution in https://github.com/arxanas/git-branchless/pull/1400
* @samueltardieu

If you're interested in contributing, check out the Developer Guide in the [Wiki](https://github.com/arxanas/git-branchless/wiki), or post in the [Discussions](https://github.com/arxanas/git-branchless/discussions) or [Discord server](https://discord.gg/caYQBJ82A4) to ask any questions.

<hr /><em>This discussion was created from the release <a href='https://github.com/arxanas/git-branchless/releases/tag/v0.10.0'>v0.10.0</a>.</em>

**Comment by KevinWuWon** [comment]
Thanks for this release! The issue that https://github.com/arxanas/git-branchless/pull/1322 fixed was driving me crazy.

**Reply by arxanas** [reply]
@jblebrun thanks for fixing it for us! It was affecting 20–30 people just based on reaction counts, and likely many more beyond that.
